src/NativeScript/inspector/GlobalObjectConsoleClient.cpp

```diff
@@ -115,12 +115,20 @@ void GlobalObjectConsoleClient::takeHeapSnapshot(JSC::ExecState*, const String&
 }
 
 void GlobalObjectConsoleClient::time(JSC::ExecState*, const String& title) {
-    m_consoleAgent->startTiming(title);
+    std::unique_ptr<String> startMsg = m_consoleAgent->startTiming(title);
+    const String* url;
```

Comment:
`url` should be declared as `String url;` and can be moved inside the `if` block as it is not used outside. The cast to `const String &` in the call to `printConsoleMessage` should be removed as well.

Reply:
Actually you can pass `WTF::emptyString()` without having to construct a new instance.